submissions/victorAienobe/phpDatabase/saveCookie.php

```diff
@@ -0,0 +1,48 @@
+<?php
+   
+    include_once('connection.php');
+    
+    //Querying the database to select all information available in the table
+    class User extends Connect {
+        public function customer() {
+            try {
+                $sql= "SELECT * FROM customer_details";
+                $stmt = $this->conn()->query($sql);
+                $result = $stmt->fetchAll();
+                
+            // setting up the cookie to save the result gotten from the databse
+            setcookie("customer", serialize($result), time()+7200, "/");
+            return $result;
+            } catch (PDOException $e) {
+                die ('could not execute ' . $sql . $e->getMessage());
+            }
+        }
+    }
+    // an instance of the class user to call the customer function
+    $now = new User();
```

Comment:
`$now` is not a good variable name that best describes this instance you just created. I think `$user` is fine